src/Field/Definition/AssetField.php

```diff
@@ -176,23 +177,28 @@ public function transformData (
 
 		if ($this->allowMultiple)
 		{
-			$transformed = [];
+			$result = [];
 
 			/** @var RawAssetData $assetDataItem */
 			foreach ($data as $assetDataItem)
 			{
-				$transformed[] = $this->transformAssetData($assetDataItem, $context);
+				$transformed = $this->transformAssetData($assetDataItem, $context);
+
+				if (null !== $transformed)
+				{
+					$result[] = $transformed;
+				}
 			}
 		}
 		else
 		{
 			/** @var RawAssetData $data */
```

Comment:
We should fix this type as it doesn't seem to exist anymore.

Reply:
I don't quite understand this point? The type is a phpstan alias, defined in the class comment.